app/views/devise/shared/_links.erb

```diff
@@ -4,8 +4,12 @@
       <%- resource_class.omniauth_providers.each do |provider| %>
         <% if provider.to_s == "google_oauth2" && (current_page?(new_user_session_path) || current_page?(new_user_password_path)) %>
           <%= button_to t('devise.shared.sign_in_google'), omniauth_authorize_path(resource_name, provider), method: :post, class: 'buttonGhostM' %>
+        <% elsif provider.to_s == "github" && (current_page?(new_user_session_path) || current_page?(new_user_password_path)) %>
```

Comment:
this part is a bit dirty, needs some refactoring

Reply:
That would be great!